payments-core/src/main/java/com/stripe/android/view/AddPaymentMethodViewModel.kt

```diff
@@ -27,23 +26,21 @@ internal class AddPaymentMethodViewModel(
         PaymentSession.PRODUCT_TOKEN.takeIf { args.isPaymentSessionActive }
     ).toSet()
 
-    internal fun createPaymentMethod(
+    internal suspend fun createPaymentMethod(
```

Comment:
I think ideally we want to model this differently. Holding this state in the UI isn't ideal. That being said. This is a net positive, and I'm peaceful with the change.